en/docs/develop/writing-a-custom-federated-authenticator.md

```diff
@@ -106,1262 +111,118 @@ public interface ApplicationAuthenticator extends Serializable {
-                    claims.put(ClaimMapping.build("screen_name", "screen_name", null,false), user.getScreenName());
-                    claims.put(ClaimMapping.build("url", "url", null,false), user.getURL());
-            
-                    context.setSubjectAttributes(claims);
-        }
-        ```
+1. Access the Management console of the partner identity server.
+2. Then go to the Service Providers under the main tab. 
+   Then add Service Provider Name and register it. 
+   Let’s use the playground app and refer to [this](https://is.docs.wso2.com/en/latest/learn/deploying-the-sample-app/#deploying-the-playground2-webapp) to configure the playground app.
+3. Then List the Service Providers and edit the service provider by navigation to the** 
+   OAuth/OpenID Configuration** under **Inbound Authentication Configuration** and add 
+   [https://localhost:9443/commonauth](https://localhost:9443/commonauth) as the callback URL.
+4. Create a user **Alex** in the partner identity server.
 
-4.  The `          buildClaims         ` method saves the retrieved user
-    attributes to the authenticated context in the Identity Server. That
-    is needed to map the claims to the built in claims of IS.
-5.  After implementing these methods you can build your bundle. After
-    building it you have to put that into the
-    `          <IS_HOME>/repository/components/dropins         `
-    directory.
-6.  [Restart the Identity Server](../../setup/running-the-product) to use the
-    Twitter authenticator in IS.
+### Configure Federated Authenticator
 
-### Custom authenticator for Facebook
+To configure the federated authenticator, click the add button under **Identity Providers** and add an IDP name as 
+**Partner-Identity-Server** and register the new IDP.
 
-This section includes the code used to write the custom authenticator
-for Facebook.
+![Partner idp config](../assets/img/using-wso2-identity-server/partner-idp-config.png)
 
-1.  Download the source code from
-    [here](https://svn.wso2.org/repos/wso2/people/isura/org.wso2.carbon.identity.application.authenticator.social/)
-    using the following command on your terminal.
+Click on the **Federated Authenticators** and expand the **custom-federated-authenticator configurations** 
+and configure it as follows.
 
-    ``` java
-    $ svn checkout https://svn.wso2.org/repos/wso2/people/isura/org.wso2.carbon.identity.application.authenticator.social/ 
-    ```
+Here, the Client Id and Client Secret are the values of external service provider from the Partner-Identity-Server.
 
-2.  Navigate to the folder you just downloaded, which contains the
-    `           pom.xml          ` file and build the source code by
-    running the following command on your terminal.
+*   _Enable / Default - You can **enable** and set to **default**_
+*   _Authorization Endpoint UR - [https://localhost:9444/oauth2/authorize/](https://localhost:9444/oauth2/authorize/)_
```

Comment:
```suggestion
*   _Authorization Endpoint UR - `https://localhost:9444/oauth2/authorize/`

Please change this wherever applicable. We avoid redirecting localhost URLs. 
```

Reply:
Fixed https://github.com/wso2/docs-is/pull/2527/commits/ffc80dd6ac9088892baa7c56e5f03a9aa0768d3f